### ISSUE TITLE
Sort tag by key

### ISSUE THREAD

**Issue by Tonic8** [markwragg/PowerShell-Influx#21]
hello

i saw that the tags are not sorted by key, wher in the doc, they say, 

For best performance you should sort tags by key before sending them to the database. The sort should match the results from the Go bytes.Compare function.

https://docs.influxdata.com/influxdb/v1.7/write_protocols/line_protocol_tutorial/

Regards
Tonic8

**Comment by Tonic8** [comment]
Hello

In Write-influx.ps1 function, add this 
```
                $TagData = $TagData -Join ','
                $TagData = ($TagData | sort-object) -Join ','
```

Regards
Tonic8

**Comment by markwragg** [comment]
Fixed by #27
